components/automate-ui/src/app/pages/+compliance/shared/reporting/report-data.service.ts

```diff
@@ -9,7 +9,10 @@ interface ReportingSummary {
     nodes: any;
     profiles: any;
     controls?: any;
+    platforms: number;
+    environments: number;
   };
+  status: 'failed' | 'passed' | 'waived' | 'skipped' | 'unknown';
```

Comment:
If you have spare cycles, would be better to make this type a reusable string literal type, so you can then strongly type other things that use it, specifically `getIcon` and `formatSummaryPhrase` in reporting.component.ts.

Reply:
There is no model file set up importing types for reporting.component.ts, would it be strange to create the interface for status in this file here (report-data.service.ts) and import it into reporting-component.ts?

Reply:
I added what I thought made the most sense.